src/js/selectbox.js

```diff
@@ -206,6 +206,17 @@ var Selectbox = (function(DX) {
 			select.addEventListener('DOMNodeInserted', optionListModificationHandler);
 			select.addEventListener('DOMNodeRemoved', optionListModificationHandler);
 		}
+		/**
+		 * Triggers when selectbox is destroyed
+		 *
+		 * @event selectbox:destroyed
+		 */
+		function destroy() {
+			block.remove();

```

Comment:
For other components (colorpicker, datepicker) you need to make the same changes